src/Card/Card.scss

```diff
@@ -34,6 +34,10 @@
 
     .pgn__card-section {
       @extend %dark-variant-content;
+
+      .pgn__card-section-title {
+        color: $white;
```

Comment:
ca we just reuse `%dark-variant-content` placeholder just like other classes above?

Reply:
done